### PR TITLE
Fixed Issue #232

### PR DESCRIPTION
**Pull request by MohamedMafaz**
Hi sir,
I created new login page
check it out

### CODE REVIEW

**Comment by QAZIMAAZARSHAD** [comment]
@MohamedMafaz host your branch on GitHub pages or anywhere else then share a live link so I can review your changes.

**Comment by MohamedMafaz** [comment]
https://www.sololearn.com/compiler-playground/W5L3wAn0Uxkl

**Comment by QAZIMAAZARSHAD** [comment]
image

The placeholder text is not going when we are typing anything.

**Comment by QAZIMAAZARSHAD** [comment]
And host the website on GitHub pages and share that link.

**Comment by MohamedMafaz** [comment]
https://mohamedmafaz.github.io/apkidukaan.github.io/

**Comment by MohamedMafaz** [comment]
Here the GitHub Page link - https://mohamedmafaz.github.io/apkidukaan.github.io/

**Comment by QAZIMAAZARSHAD** [comment]
image

Update the title -> Animated Login Form -> Sign In
Update the image on the left side -> It Should be related to e-commerce or the logo of Apni Dukaan
Align the position of the Sign-Up link.
Add a hyperlink to forget password page on forget password link.
Rename it "Log in and use the opportunities offered by our portal." -> Exciting deals waiting for you

When I go to sign-up page and return back to the login page -> it shows like this -> error in placeholder text -> fix it
image

When I enter ggg@j or ggg@gmail.c in the email field and press login -> the page reloads and inputs vanish -> It should give an error to edit the email field

And when the email field is correct and password is empty -> it should give error "Please enter password" but page is getting refreshed

Fix these things.

And validate the password field as well -> it should accept a password whose length should be between 8-15 characters and must contain at least 1 capital and 1 lower case letter, at least 1 special character, at least 1 number.

**Comment by MohamedMafaz** [comment]
https://mohamedmafaz.github.io/apkidukaan.github.io/ - Here the Updated Page link
But I don't know how to restrict password. I try many times but it's not working. so I don't do that

**Comment by MohamedMafaz** [comment]
https://mohamedmafaz.github.io/apkidukaan.github.io/index.html? - Here the Updated Page check it out

File: login.html [hunk 1]
```diff
@@ -1,45 +1,49 @@
...
+           		    	<i class="fas fa-lock"></i>
+           		   </div>
+           		   <div class="div">
+           		    	<h5>Password</h5>
+           		    	<input type="password" class="input">
+            	   </div>
+            	</div>
+            	<a href="#">Forgot Password?</a>
+            	<input type="submit" class="btn" value="Login">
+              <p>Don't have an account</p><a href="https://qazimaazarshad.github.io/Apni-Dukaan/signup.html">Sign Up</a>
```
Comment (QAZIMAAZARSHAD) [inline login.html:41]: Don't use links like this. 
Redirect it to the signup page using this way "/folderfilelocation".

Update links everywhere where you have done this.

**Comment by QAZIMAAZARSHAD** [comment]
Replace this line "Don't have an account" with "Don't have an account! Sign Up"

**Comment by QAZIMAAZARSHAD** [comment]
Change Green color to blue color so it matches the website theme.

**Comment by MohamedMafaz** [comment]
https://mohamedmafaz.github.io/apkidukaan.github.io/index.html Check it out

**Comment by QAZIMAAZARSHAD** [comment]
Still same @MohamedMafaz you have not pushed your changes to your branch.

https://mohamedmafaz.github.io/apkidukaan.github.io/index.html  not updated here.

**Comment by QAZIMAAZARSHAD** [comment]
Also, update the favicon as well. It is not visible.

**Comment by MohamedMafaz** [comment]
https://mohamedmafaz.github.io/apkidukaan.github.io/

**Comment by QAZIMAAZARSHAD** [comment]
nope.

Still green.
Links not working.
Favicon not visible.

**Pull request closed without merging**